src/fast_binomial.inl

```diff
@@ -44,7 +44,7 @@ inline FastBinomialFixed<is_scalar_p, CacheSize, PRNG>::FastBinomialFixed(
 }
 
 template<bool is_scalar_p, unsigned short CacheSize, typename PRNG>
-inline FastBinomialFixed<is_scalar_p, CacheSize, PRNG>::value_type
+inline typename FastBinomialFixed<is_scalar_p, CacheSize, PRNG>::value_type
```

Comment:
Needed change due to downgraded c++ standard (20 -> 17). Making it more compatible with older compilers.